src/main/app/src/components/YhteystiedotSection.tsx

```diff
@@ -0,0 +1,142 @@
+import React, { useCallback } from 'react';
+
+import { useTranslation } from 'react-i18next';
+import { Field, FieldArray } from 'redux-form';
+
+import {
+  FormFieldInput,
+  FormFieldPostinumeroSelect,
+  FormFieldUrlInput,
+} from '#/src/components/formFields';
+import { Divider } from '#/src/components/virkailija';
+import { getTestIdProps } from '#/src/utils';
+
+import { Button } from './Button';
+import { FieldArrayList } from './FieldArrayList';
+import { Flex } from './Flex';
+import { GridColumn } from './GridColumn';
+import { GridRow } from './GridRow';
+
+export const YhteystietoSection = ({ description, name, language }) => {
+  const { t } = useTranslation();
+
+  return (
+    <GridRow>
+      {description && (
+        <GridColumn css={'margin-top: 24px'} md={12}>
+          {description}
+        </GridColumn>
+      )}
+      <GridColumn css={'margin-top: 24px'} md={12} {...getTestIdProps('nimi')}>
```

Comment:
Näitä test-id:itä ei mun mielestä pitäisi tarvita. Cypressissä pitäisi pystyä hyvin käyttämään esim. `cy.findByLabelText('yleiset.postiosoite')`.